src/tasks/dump-db.toml

```diff
@@ -0,0 +1,168 @@
+# Column visibility in database dumps. Only public columns are included in the dump.
+[api_tokens]
+id = "private"
```

Comment:
Do we need to specify `private` here? This should 100% for sure be an allow-list, so everything needs to be private by default anyway.

Reply:
The idea is to have visibility information for all columns. There is a unit test that compares the schema of the test database to the configuration in this file, so when a new column is added, people are explicitly forced to decide whether it should be public or private. I arrived at this design in response to https://github.com/rust-lang/crates.io/issues/630#issuecomment-510642673.